Dockerfile

```diff
@@ -1,17 +1,14 @@
-FROM ubuntu:15.10
-RUN apt-get update && \
+FROM ubuntu:16.04
+RUN apt-get update && apt-get install -y build-essential wget curl iproute2 && \
     apt-get build-dep -y strongswan
-RUN apt-get install -y wget curl iproute2
 
-ARG VERSION=5.3.5
+ARG VERSION=5.4.0
 RUN wget https://download.strongswan.org/strongswan-${VERSION}.tar.bz2
 RUN tar xvf strongswan-${VERSION}.tar.bz2
 WORKDIR strongswan-${VERSION}
 RUN ./configure \
-    --enable-swanctl \
```

Comment:
swanctl and vici are included by default in strongswan-5.4, so yeah, the options are dropped.